release-tools/build.make

```diff
@@ -63,6 +63,7 @@ endif
 build-%:
 	mkdir -p bin
 	CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-X main.version=$(REV) -extldflags "-static"' -o ./bin/$* ./cmd/$*
+	CGO_ENABLED=0 GOOS=windows go build -a -ldflags '-X main.version=$(REV) -extldflags "-static"' -o ./bin/$*.exe ./cmd/$*
```

Comment:
do we also need to modify container and push rules?

Reply:
Yes. That is the part that needs a bit of discussion and research to get a Windows build platform in the picture since the Windows containers can only be built on Windows hosts.